buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -927,19 +739,24 @@ class BuildPlugin implements Plugin<Project> {
                     jarTask.manifest.getAttributes().clear()
                 }
             }
+
             // add license/notice files
             project.afterEvaluate {
-                if (project.licenseFile == null || project.noticeFile == null) {
+                if (ext.has('licenseFile') == false || ext.get('licenseFile') == null || ext.has('noticeFile') == false || ext.get('noticeFile') == null) {
```

Comment:
This seems to have broken our custom plugin builds. The same build.gradle file which worked for all 6.x and 7.x builds with esplugin is now broken and fails with

```
* What went wrong:
A problem occurred configuring root project 'elasticsearch-analysis-hebrew'.
> Failed to notify project evaluation listener.
   > Must specify license and notice file for project :
   > Must specify license and notice file for project :
   > Must specify license and notice file for project :
```

It is indeed possible that I'm missing something, but we do have noticeFile and licenseFile properly declared and moving them from project to ext to esplugin doesn't work.

Any idea?

Reply:
OK n/m just found https://github.com/elastic/elasticsearch/issues/45073#issuecomment-517388642 , which fixes this issue